sapi/cli/tests/cli_set_process_title_basic.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+cli_set_process_title() function : basic functionality
+--CREDITS--
+Patrick Allaert patrickallaert@php.net
+@nephp #nephp17
+--SKIPIF--
+<?php
+if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN')
+  die("skip");
+?>
+--FILE--
+<?php
+if (cli_set_process_title("title") === true &&
```

Comment:
There is an overlap, for sure, this one focuses on cli_set_process_title(), hence checks the return result which is not the case in the other one, which focuses more on cli_get_process_title().